website/templates/index.html

```diff
@@ -35,12 +35,10 @@ <h4 class="card-title">{% trans "Number Error" %}</h4>
                       </div>
                     </div>
                     <ul class="list-group list-group-flush">
-                        <li class="list-group-item">
-                            <a href="/search/?query=1&type=label">Examples</a>
-                        </li>
+                        <a class="btn btn-default" href="/search/?query=1&type=label">Examples</a>
                     </ul>   
                 </div>
-                <div class="col-sm-4">    
+                <div class="col-sm-3 cborder">
```

Comment:
I can add that way but in other screen sizes i thought it may overflow. I will try will sm-4 and check again with different method.